src/vma-redirect.h

```diff
@@ -33,7 +33,8 @@
 #include <unistd.h>
 #include <stdlib.h>
 #include <signal.h>
-#include <sys/poll.h>
+#include <stdio.h>
```

Comment:
Why is `stdio.h` being included to `vma-redirect.h`? Not clear to me.

Reply:
Same here, this is another change required to build sockperf in alpine OS